src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/Email
```

Comment:
Consider using YAML, as it is easier to read and offers tree-like structure for properties